src/explore-education-statistics-frontend/src/modules/find-statistics/PublicationReleasePage.tsx

```diff
@@ -125,6 +125,7 @@ const PublicationReleasePage: NextPage<Props> = ({ release }) => {
                 <Details
                   id="releaseLastUpdates"
                   summary={`See all updates (${updates.length})`}
+                  hiddenText={`for ${release.publication.title}`}
```

Comment:
Updates aren't for a publication, they're for a release. We should change this to `for ${release.title}`.